app/models/chargeback.rb

```diff
@@ -8,19 +8,10 @@ class Chargeback < ActsAsArModel
 
   def self.build_results_for_report_chargeback(options)
     _log.info("Calculating chargeback costs...")
+    @options = options = ReportOptions.new_from_h(options)
```

Comment:
why there is also used `options` when we have`@options` ?  even when `options` is rewrited.